addons/css-fix/userstyle.css

```diff
@@ -0,0 +1,18 @@
+
+.blocklyWidgetDiv .goog-menu {
+overflow: hidden; /* Hide scrollbars */
+}
+.sprite-selector_items-wrapper_4bcOj.box_box_2jjDp {
```

Comment:
The hash at the end should not be included (use something like `[class^="sprite-selector_items-wrapper_"]`) for compatibility with TurboWarp Desktop.